lib/cchain/provider/xblock.go

```diff
@@ -59,7 +59,7 @@ func toPortalVals(vals []cchain.Validator) []bindings.Validator {
 	for i, val := range vals {
 		portalVals[i] = bindings.Validator{
 			Addr:  val.Address,
-			Power: val.Power,
+			Power: uint64(val.Power),
```

Comment:
Need to be careful with negatives here right? So we don't give validators slashed to negative power a ton of voting power on chain.

- could `val.Power` be negative? if not, maybe an error here if it is
- if it could be, let's either trim to 0, not include it in the set, or use `int64` on chain
     - if we trim to 0, we'll need to allow 0 power on chain (right now [we don't](https://github.com/omni-network/omni/blob/main/contracts/src/protocol/OmniPortal.sol#L281))